### PR TITLE
Added new value to HostName enum for T21

### PR DESCRIPTION
**Pull request by HarshJain27**
For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description

Added new value to HostName enum : To identify Teams v1 (Built using electron+angular) Vs Teams v2.1 (Built using WebView2+react)

Design Document:
https://microsoft.sharepoint.com/:w:/t/whatsupdoc/ERpFucwyimxEvQ9pz8y93kMBW2YkEkrzwpBM77i3hTZbwA?e=QcIboB

### Change file added:

> Ensure the change file meets the [formatting requirements](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md#change-log-using-beachball).

Yes

### Related PRs:

[Pull Request 1821458](https://office.visualstudio.com/ISS/_git/metaos-hub-sdk/pullrequest/1821458): Added new value to HubNames enum : To identify Teams v1 Vs Teams v2.1

### CODE REVIEW

File: packages/teams-js/src/public/constants.ts [hunk 1]
```diff
@@ -42,6 +42,11 @@ export enum HostName {
    * Teams
    */
   teams = 'Teams',
+
+  /**
+   * Teams 2.1
```
Comment (TrevorJoelHarris) [inline packages/teams-js/src/public/constants.ts:47]: This comment will get turned into user facing documentation. Are you okay with "Teams 2.1" being how users think of this?

**Review by TrevorJoelHarris — APPROVED** [review]

File: packages/teams-js/src/public/constants.ts (hunk 1 above)
Comment (HarshJain27) [reply packages/teams-js/src/public/constants.ts]: Thanks for pointing this out. Have changed the value to `Modern Teams`.

**Review by TrevorJoelHarris — APPROVED** [review]

**Review by AE-MS — APPROVED** [review]
:shipit:

**Pull request merged**